### PR TITLE
[IOPAY-136] new project for io-pay

### PR DESCRIPTION
**Pull request by infantesimone**
### Description

This PR adds new project for [io-pay](https://github.com/pagopa/io-pay).

### List of changes

- new folder for the new _io-pay_ project with basic configuration;
- new _io-pay.tf_  file with _code-review_ and _deploy_ pipelines with related variables.

### CODE REVIEW

File: azure-devops/projects/io-pay-projects/service_connections.tf [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+
+# npm service connection
+resource "azuredevops_serviceendpoint_npm" "pagopa-npm-bot" {
+  depends_on = [azuredevops_project.project]
+
+  project_id            = azuredevops_project.project.id
+  service_endpoint_name = "pagopa-npm-bot"
+  url                   = "https://registry.npmjs.org"
+  access_token          = data.azurerm_key_vault_secret.key_vault_secret["pagopa-npm-bot-TOKEN"].value
+}
```
Comment (pasqualedevita) [inline azure-devops/projects/io-pay-projects/service_connections.tf:89]:
```suggestion
```
@infantesimone do u need to connect to npm registry to publish something? if u don't need it delete this please

File: azure-devops/projects/io-pay-projects/secrets.tf [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+variable "secrets" {
+  default = [
+    "DANGER-GITHUB-API-TOKEN",
+    "io-azure-devops-github-ro-TOKEN",
+    "io-azure-devops-github-rw-TOKEN",
+    "io-azure-devops-github-pr-TOKEN",
+    "io-azure-devops-github-EMAIL",
+    "io-azure-devops-github-USERNAME",
+    "pagopa-npm-bot-TOKEN",
```
Comment (pasqualedevita) [inline azure-devops/projects/io-pay-projects/secrets.tf:13]:
```suggestion
```
@infantesimone do u need to connect to npm registry to publish something? if u don't need it delete this please

**Review by pasqualedevita — CHANGES_REQUESTED** [review]

**Comment by pasqualedevita** [comment]
https://github.com/pagopa/io-pay/blob/master/.devops/deploy-pipelines.yml#L35

Also if possibile change this user
`pagopa` -> `io-azure-devops-github-ro`

File: azure-devops/projects/io-pay-projects/service_connections.tf (hunk 1 above)
Comment (infantesimone) [reply azure-devops/projects/io-pay-projects/service_connections.tf:89]: no, thanks

**Comment by infantesimone** [comment]
so can I delete `"azuredevops_serviceendpoint_github" "pagopa"` and use `io-azure-devops-github-ro ` in pipelines?

**Comment by pasqualedevita** [comment]
> ok, I will use `io-azure-devops-github-ro` in pipelines.

thank u Simone

**Review by pasqualedevita — COMMENTED** [review]
LGTM
@infantesimone I'll add a section to give correct role into Azure. It's my task

**Review by pasqualedevita — APPROVED** [review]
added `Storage Blob Data Contributor` role assignment

**Comment by pasqualedevita** [comment]
all ok :) thanks @infantesimone

**Pull request merged**